products/green_fast_track/models/intermediate/buffers/int_buffers__all.sql

```diff
@@ -46,6 +46,7 @@ WITH all_buffers AS (
         ref('int_buffers__lpc_landmarks'),
         ref('stg__nysshpo_archaeological_buffer_areas')
     ],
+    include=["variable_type", "variable_id", "raw_geom", "buffer"],
```

Comment:
What does this do?

Reply:
just noting for later:

didn't know this util creates a column called `_dbt_source_relation` "to indicate the source for each record"!! seems like a cool way to avoid declaring `variable_type` strings in these models and just use the model names to drive queries??

Reply:
Ooh that's interesting. Love that